cfme/automate/explorer/domain.py

```diff
@@ -375,11 +376,9 @@ def all(self):
                     .query(repo_table)\
                     .filter(repo_table.id == git_repository_id)\
                     .first()
-                from cfme.automate.import_export import AutomateGitRepository
-                agr = AutomateGitRepository(
-                    url=repo.url,
-                    verify_ssl=repo.verify_ssl,
-                    appliance=self.appliance)
+                agr = self.appliance.collections.automate_import_export.instantiate(
```

Comment:
The collection is different here. `automate_import_export` vs `automate_import_exports` above.

Reply:
aaaaaaaa **Eagle Eyes**. I missed this while adding changes from shveta comment :).